tests/extensions/sde009-safe_test.ts

```diff
@@ -0,0 +1,122 @@
+import { 
+  Account,
+  Clarinet,
+  Chain,
+  types,
+} from '../utils/helpers.ts';
+import { ExecutorDao } from '../models/executor-dao-model.ts';
+import { SDE006Membership, SDE006_MEMBERSHIP_CODES } from '../models/sde006-membership-model.ts';
+import { SDE007ProposalVoting, SDE007_PROPOSAL_VOTING_CODES } from '../models/sde007-proposal-voting-model.ts';
+import { SDE008ProposalSubmission, SDE008_PROPOSAL_SUBMISSION_CODES } from '../models/sde008-proposal-submission-model.ts';
+import { SDE009Safe, SAFE_CODES } from '../models/sde009-safe-model.ts'; 
+import { PROPOSALS, EXTENSIONS, TEST_EXTENSIONS, TEST_PROPOSALS } from '../utils/contract-addresses.ts';
+
+
+Clarinet.test({
+  name: '😢 SDE009Safe - Token not whitelisted',
```

Comment:
this doesnt really check if a token is not whitelisted, this is an authentication check on who is allowed to call setWhitelisted

Reply:
sdp007-whitelist-asset was an old file that was removed.   Removed reference in Clarinet.toml

In /tests/proposals, sdp009-test-whitelist-asset is the current file as an 008 exists already.  Was trying to discern/follow your paradigm there.  Can do either of the below:
1) Move it out of tests and rename with sdp008...
2) Leave in tests as 009

Moved current /tesets/proposals 009 to be below 008 in Clarinet.toml

Reply:
SD009Safe - Token not whitelisted: 
Modified name of test to more accurately reflect what it is testing for.

Added actual whitelist check at beginning of whitelist asset testing to confirm asset is not initially whitelisted.